tests/wicked/aggregate/sut/t11_teaming_roundrobin.pm

```diff
@@ -0,0 +1,43 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Teaming, roundrobin
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+#             Jose Lausuch <jalausuch@suse.com>
+#             Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+use Mojo::Base 'wickedbase';
+use utils;
+use network_utils 'ifc_exists';
+use testapi;
+
+
+sub run {
+    my ($self, $ctx) = @_;
+
+    my $cfg_team0 = '/etc/sysconfig/network/ifcfg-team0';
+    my $cfg_ifc0  = '/etc/sysconfig/network/ifcfg-' . $ctx->iface();
+    my $cfg_ifc1  = '/etc/sysconfig/network/ifcfg-' . $ctx->iface2();
+
+    zypper_call('-q in libteam-tools libteamdctl0 python-libteam');
+
+    $self->get_from_data('wicked/teaming/ifcfg-eth0',             $cfg_ifc0);
+    $self->get_from_data('wicked/teaming/ifcfg-eth1',             $cfg_ifc1);
+    $self->get_from_data('wicked/teaming/ifcfg-team0-roundrobin', $cfg_team0);
+
+    $self->wicked_command('ifup', 'team0');
+    ifc_exists('team0');
+    validate_script_output('ip a s dev ' . $ctx->iface(),  sub { /master team0/ });
+    validate_script_output('ip a s dev ' . $ctx->iface2(), sub { /master team0/ });
+
+    $self->ping_with_timeout(type => 'host', interface => 'team0');
```

Comment:
this two lines , copy/pasted in 3 tests up to now ( this one + 2 flavors of aggregate both cards ) maybe it is the time to extract it to a function ?

Reply:
Hm, I think you mix it with `SLAVE` for binding.
But yes, we could make it as a utility function.

Reply:
Sorry I misunderstood you. I thought you are speaking about the `ip a s dev | grep master` lines.
Put the code inside of `ping_with_timeout()`